src/js/renderShabad.js

```diff
@@ -91,6 +91,27 @@ function metaData(data, nav) {
 }
 
 function Baani(gurbani) {
+  const onCopyClick = ({ currentTarget }) => {
+    const textarea = currentTarget.parentNode.parentNode.querySelector('textarea');
+    textarea.style.display = 'block';
+    textarea.focus();
+    textarea.select();
+    document.execCommand('copy');
+    textarea.blur();
+    textarea.style.display = 'none';
```

Comment:
Not sure if changing `style.display` is needed.

Reply:
I believe that was put in because it otherwise didn't perform the select+copy